### DISCUSSION TITLE
绘制区域覆盖到海里就不见了，陆地上没有问题

### DISCUSSION THREAD

**Discussion by layogo** [zouyaoji/vue-cesium#532] [General]
![image](https://github.com/zouyaoji/vue-cesium/assets/674578/e28c0728-b541-4aa9-9a9b-f3e984d824a7)

**Comment by layogo** [comment]
![image](https://github.com/zouyaoji/vue-cesium/assets/674578/703203ee-f778-46e9-aeff-5c48c411290f)

**Reply by zouyaoji** [reply]
我用文档的例子尝试了一下，没能够重现，你开启下面这句没？

viewer.scene.globe.depthTestAgainstTerrain = true

**Reply by layogo** [reply]
你看看台湾下面那片海域，图形框选面积大了就不会，小了会

**Reply by layogo** [reply]
`viewer.scene.globe.depthTestAgainstTerrain = true`
这个是设置了的，可能跟地形数据有点关系吗
